companies/tasks/dart_sync.py

```
@@ -157,11 +159,64 @@ def sync_all_company_info():
+                # 현재부터 과거 N년치 재무제표 동기화 작업 등록
+                for year_offset in range(years):
+                    target_year = current_year - year_offset
+                    # countdown을 사용하여 작업을 시간차로 분산
+                    countdown_offset = (i // batch_size) * 60 + year_offset * 10
+                    task = sync_financial_statements.apply_async(
+                        args=[company.stock_code, target_year],
+                        countdown=countdown_offset
+                    )
+                    tasks.append(task)
+                    task_count += 1
+            except (Reject, ConnectionError) as e:
```

Ruff (0.14.13)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)